config.go

```diff
@@ -9,11 +9,35 @@ import (
 
 // Configuration is a minimal set of parameters for Sentry integration.
 type Configuration struct {
-	Tags              map[string]string
+	// Tags are passed as is to the corresponding sentry.Event field.
+	Tags map[string]string
+
+	// LoggerNameKey is the key for zap logger name.
+	// If not empty, the name is added to the rest of zapcore.Field(s),
+	// so that be careful with key duplicates.
+	// Leave LoggerNameKey empty to disable the feature.
+	LoggerNameKey string
+
+	// DisableStacktrace disables adding stacktrace to sentry.Event, if set.
 	DisableStacktrace bool
-	Level             zapcore.Level
-	BreadcrumbLevel   zapcore.Level
-	EnableBreadcrumbs bool // this feature works only when you explicitly passed new scope
-	FlushTimeout      time.Duration
-	Hub               *sentry.Hub
+
+	// Level is the minimal level of sentry.Event(s).
+	Level zapcore.Level
+
+	// BreadcrumbLevel is the minimal level of sentry.Breadcrumb(s).
+	// Breadcrumb specifies an application event that occurred before a Sentry event.
+	// NewCore fails if BreadcrumbLevel is greater than Level.
+	// The field is ignored, if EnableBreadcrumbs is not set.
+	BreadcrumbLevel zapcore.Level
+
+	// EnableBreadcrumbs enables use of sentry.Breadcrumb(s).
+	// This feature works only when you explicitly passed new scope.
+	EnableBreadcrumbs bool
```

Comment:
Maybe it's just me, but I would move `EnableBreadcrumbs` before the `BreadcrumbLevel`...just a note :)

Reply:
I left the order of the fields untouched, just commented.
I agree, it is probably better to move the field up.